controllers/curriculum/curriculum.go

```diff
@@ -0,0 +1,62 @@
+package curriculum
+
+import (
+	"tat_gogogo/crawler/curriculum"
+
+	"log"
+
+	"github.com/gin-gonic/gin"
+)
+
+/*
+Controller handle curriculum login api
+*/
+func Controller(c *gin.Context) {
+	studentID := c.PostForm("studentId")
```

Comment:
I think we can unify all the naming convention with ID

Reply:
cause we gotta discuss about the naming convention between outside/inside of this project.
i.e. when front-end project use this api, do we except that the developer use `studentID ` instead of `studentId `, if so, we could change all of it to `studentID`

Reply:
I think we can just unify it as `studentID`, and it can reduce errors in maintenance